drivers/pwm/Kconfig

```diff
@@ -21,6 +21,12 @@ config PWM_SHELL
 	help
 	  Enable the PWM related shell commands.
 
+config PWM_CAPTURE
+	bool "Provide API for PWM capture"
```

Comment:
I would prefer not to introduce that. I find it highly confusing as a user when a given API cannot be enabled in Kconfig due to a "hidden" setting like that. What do others think?

Reply:
It just occurred to me that introducing a `CONFIG_PWM_SUPPORT_CAPTURE` or similar Kconfig option will not work well for PWM since there are PWM capture-only devices out there (see e.g. the NXP Kinetis Pulse Width Timer (PWT) in #31165), for which the driver Kconfig option itself will depend on `CONFIG_PWM_CAPTURE`. This will introduce circular dependencies.

Reply:
The consensus on todays API meeting was to go without a `CONFIG_PWM_SUPPORT_CAPTURE` Kconfig option, at least for now.